src/storage/hummock_sdk/src/frontend_version.rs

```diff
@@ -161,8 +162,8 @@ impl FrontendHummockVersionDelta {
                             truncate_epoch: change_log_delta.truncate_epoch,
                             new_log: change_log_delta.new_log.as_ref().map(|new_log| {
                                 EpochNewChangeLogCommon {
-                                    new_value: vec![],
-                                    old_value: vec![],
+                                    new_value: new_log.new_value.iter().map(|_| ()).collect(),
```

Comment:
`vec![(); new_log.new_value.len()]`

And the `old_value` incorrectly use the `new_log.new_value`.